### ISSUE TITLE
[Suggestion] - Mod Compatibility - Multiple (Explained below)

### ISSUE THREAD

**Issue by XvBUSHvX** [MightyPirates/Vials#11]
The following was a comment I had intended to further a mod compatibility conversation that was posted on the curseforge project page for your Vials mod. Upon submitting the comment it was instantly marked as spam with a red background. I know that it was a rather lengthy comment, and suspect that is why it was flagged. I hope that you read it and find it to be a worth while post. (Edited to fix spelling and grammatical errors)

> 
> If I may contribute my own two cents here. I too have been quite fond of the different sized vials, even though TiC has fallen out of favor for me personally I would still like to find a use for them and their novel intricacies.
> 
> Though trivial in many aspects, the ability to hold varying degrees or more mundane fluids would open up a lot of potential for mod pack makers for recipes. a "Pinch of this, dash of that approach" if you would. with some ingredients producing more product. With "CraftTweaker" some item recipes could become more intricate than tedious in mod packs.
> 
> To further expand though, I have always found it odd that a vanilla bucket and glass bottle share the same volume.  And beyond that just how much milk do you have to chug to remove affects? I would think a much smaller portion would suffice. Milk, does a body good.
> 
> I honestly haven't had the opportunity to experience with the "Extra Alchemy" mod. It is in my pool of mods I'm going to begin testing in the near future and it seems to offer a lot potions. Indeed it even offers it's own type of 'vial'.
> 
> Though as it is described on the project page his vials are simply intended to provide a stacking version of the vanilla potion bottle. However, he has a method to combine potions of the same type and amplifier to increase the duration. This can this be reversed by crafting the potion bottle/vial back together with empty bottles/vials to divide the the duration of the potions among the provided bottles.
> 
> Something that I could see adding to the immersion of the process would be to craft potions in one of your 1000mB vials then dividing it among the smaller ones granting the same amplifier of the potion brewed but dividing the duration based on the volume of potion in the vial. Rounding down for the duration cause, you know, spillage when pouring into a smaller bottle.
> 
> Combining potion effects?!?! Whoa boy, well obviously you would be able to increase duration by mixing two like potions into a larger Vial, that's just....logical. But factor in compatibility with the "Combined Potions" mod. Imagine not only increasing duration by blending like potions with like, but combining two, or more, for different effect and increased duration. Optionally mixing different amplifiers for a "chance" to increase the affect level at a cost to the duration.
> 
> And if that ain't enough potiony fun, well then that's just "Not Enough Potions" (mod), but we'll get down to the "Potion Core". When your big ol vial can be used to serve specially made potions in that big ol cauldron they've got. But you gotta be careful can't drink too much cause you'll end up getting potion sickness and liable to blow yourself apart.
> 
> ----------------------------
> 
> Now I know that this isn't something that you can do alone. But seriously, I think that between you and the other 3 devs of the mods I listed, a wickedly bad a$$ set of mods that would greatly expand on what's basically vanilla minecraft would be achievable.
> 
> A combination that could really influence the early game. Modpackers could increase the threat of mobs, while at the same time offering the players an immersive way to counteract the risk.
> 
> I'm new, extremely new, to assembling mod packs. Have yet to have even made one yet, but, I've been reading through mods to assemble a pool of mods I'd like to use. And I'm nearly at the end of the list before I move beyond that step, the letter "V" to be precise. Over the past week I've been Inspired by a number of mods and look forward to seeing if I can make what I imagine possible. 
> 
> What I have written in this comment, it's a book I know and apologize, is the inspiration I felt think about this mod and the others. And I hope the excitement I have expressed inspires you as much, to contact the other developers, and that they become just as inspired because it would take all of your mods to accomplish this idea.

**Comment by lordjoda** [comment]
While your suggestions sound very interesting, I fear that neither I have the time to implement them myself, nor do I think that (at least most of them) would fit in the general "idea" of this mod. Therefore a new Mod or the integration in the other mods would make more sense IMHO.

**Comment by XvBUSHvX** [comment]
I agree totally. I would believe that for the most part it is something the other devs would have to implement in their mods. An API to facilitate is all I could see you needing to incorporate. Maybe something that would dynamically create bottles with fluids like the forge buckets. I sadly don't code so I really don't have any background to understand how difficult that would be. I would be more than happy, with your consent, to call the other devs attention to this thread to initiate a conversation. It would give the opportunity for everyone to see if it's possible and what it would require from each person.
